### DISCUSSION TITLE
Publii-lite viewer?

### DISCUSSION THREAD

**Discussion by RLO1729** [GetPublii/Publii#1850] [Ideas]
Hi, I've built a Publii website that I would like to share with individuals off-line rather than have it hosted somewhere. I could ask them to install Publii and tell them which directory to upload my website files to but some of them won't be able to do that. It would be useful to have a stand-alone "Publii-lite" viewer-only version (just the preview part of Publii) for them to see the site as I do from within Publii. Or could Publii also generate a stand-alone exe version of the site? Thanks.

**Comment by shawngiese** [comment]
I think you can just output the files to a folder and zip that up.

**Reply by RLO1729** [reply]
Thanks, tried that but the webpage links all refer to my computer's directories, which don't work on someone else's computer.

**Reply by shawngiese** [reply]
Does that happen when you set Server > Zip > Use Relative URL? If they / you want to serve it you could have them install Python and run "python -m http.server".

**Reply by RLO1729** [reply]
Thanks! Using relative URL works - almost. After unzipping the file, clicking on index.html opens the Home webpage and all the links work except the Home menu link which opens a text directory listing, not the Home page.

**Reply by RLO1729** [reply]
So hopefully I can just not set a Home page and adjust the menu. Will give it a go.

**Reply by RLO1729** [reply]
Problem solved - thanks again!

**Reply by RLO1729** [reply]
Everything works except clicking on the website logo - which opens the text directory listing - but I can live with that.

**Reply by Thorev** [reply]
> Everything works except clicking on the website logo - which opens the text directory listing - but I can live with that.

Try this option. It might help.
![Screen2024-12-06 023215](https://github.com/user-attachments/assets/2f0d7751-2c9c-4068-a948-9459d13e110b)

**Reply by RLO1729** [reply]
Yes it did thanks. Now clicking on the logo opens the Hero section (which I wasn't using on this website) but much better than the text directory listing!

**Reply by RLO1729** [reply]
Great, edited the Hero section to suit. Now I have a professional looking and fully-functioning package to share with others. I suggest instructions for creating a zip file to share with others along the lines of this discussion be added to the Publii documentation on servers.
